frontend/src/components/LandlordCard/LandlordCard.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { ReactElement } from 'react';
+import ApartmentImg from '../../assets/apartment-placeholder.png';
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Grid,
+  makeStyles,
+  Typography,
+  useMediaQuery,
+} from '@material-ui/core';
+import { LandlordWithLabel } from '../../../../common/types/db-types';
+import { colors } from '../../colors';
+
+type Props = {
+  landlordData: LandlordWithLabel;
+};
+const useStyles = makeStyles({
+  img: {
+    paddingTop: '10.25%',
+    borderRadius: '50%',
+    paddingRight: '5px',
+    margin: '10px',
+    width: '100%',
+  },
+  landlordNameTxt: {
+    color: colors.white,
+    fontWeight: 600,
+    marginLeft: '25px',
+  },
+  marginTxt: {
+    marginLeft: '25px',
+  },
+  card: {
+    height: '100%',
+    borderRadius: '10px',
+    backgroundColor: colors.landlordCardRed,
+    paddingTop: '10px',
+  },
+});
+
+const LandlordCard = ({ landlordData }: Props): ReactElement => {
+  const classes = useStyles();
```

Comment:
You can actually destructure this so that you don't have to keep using `classes.`! 
```
const {card, marginTxt, ... } = useStyles();
```